e107_plugins/social/plugin.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<e107Plugin name="Social" version="1.0" date="2012-12-01" compatibility="2.0" installRequired="true" >
+<e107Plugin name="LAN_PLUGIN_SOCIAL_NAME" version="1.0" date="2012-12-01" compatibility="2.0" installRequired="true" >
 	<category>settings</category>
```

Comment:
This is incorrect.  The correct way is: 
`<e107Plugin name="Social" lan="LAN_PLUGIN_SOCIAL_NAME" ....`